playing_game.py

```diff
@@ -0,0 +1,43 @@
+#!/bin/python3
+
+import os
+import sys
+
+#
+# Complete the gamingArray function below.
+#
+def gamingArray(arr):

```

Comment:
This is only the first round of the game. The game is played g times with Bob starting the first round then they alternate.

Reply:
[playing_game.txt](https://github.com/priyaSNHU/Problem-Solving/files/1934403/playing_game.txt)

Reply:
can you please review the problem, i have made all required corrections in it.